File: back/src/models/Job.js
import mongoose from 'mongoose';

const jobSchema = new mongoose.Schema(
  {
    id: { type: String, index: true },
    title: { type: String, default: '' },
    name: { type: String, default: '' },
    order: { type: Number, default: 0 },
    serviceYear: { type: String, default: '2026', index: true },
  },
  {
    timestamps: true,
    strict: false,
    collection: 'jobs',
  }
);

const Job = mongoose.models.Job || mongoose.model('Job', jobSchema);
export default Job;

File: back/Models/Job.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Eftekad.Backend.Models
{
    [Table("jobs")]
    public class Job
    {
        [Key]
        [Column("id")]
        public string Id { get; set; } = string.Empty;

        [Column("title")]
        public string Title { get; set; } = string.Empty;

        [Column("name")]
        public string Name { get; set; } = string.Empty;

        [Column("order")]
        public int Order { get; set; } = 0;

        [Column("serviceYear")]
        public string ServiceYear { get; set; } = "2026";

        [Column("createdAt")]
        public DateTime CreatedAt { get; set; } = DateTime.UtcNow;

        [Column("updatedAt")]
        public DateTime UpdatedAt { get; set; } = DateTime.UtcNow;
    }
}
